plugins/alerts/frontend/public/javascripts/countly.views.js

```diff
@@ -12,811 +12,483 @@
+                this.resetAlertCondition();
             }
-            $("#alert-compare-value-input").attr("placeholder", jQuery.i18n.map["alert.add-number"]);
         },
-        init: function() {
-            var self = this;
-            // clear alertName
-            $("#alert-name-input").val('');
-            $("#alert-name-input").off('input').on('input', function() {
-                self.checkDisabled();
-            });
-            // select alert data type : metric , event crash
-            var metricClickListner = function() {
-                $("#single-target-dropdown").off("cly-select-change").on("cly-select-change", function(e, selected) {
-                    var dataType = $(($('#alert-data-types').find(".selected")[0])).data("dataType");
-                    var source = $("#" + dataType + "-condition-template").html();
-                    $('.alert-condition-block').html(source);
-
-                    if (selected === 'Number of page views') {
-                        source = $("#metric2-condition-template").html();
-                        $('.alert-condition-block').html(source);
-                        $("#single-target-dropdown").clySelectSetItems(alertDefine[dataType].target);
-                        self.loadAppViewData();
-                    }
-                    else if (selected === 'Number of ratings') {
-                        source = $("#rating-condition-template").html();
-                        $('.alert-condition-block').html(source);
-                        $("#single-target-dropdown").clySelectSetItems(alertDefine[dataType].target);
-                        self.loadRatingOptions();
-                    }
-                    else if (selected === 'Bounce rate') {
-                        source = $("#metric2-condition-template").html();
-                        $('.alert-condition-block').html(source);
-                        $("#single-target-dropdown").clySelectSetItems(alertDefine[dataType].target);
-                        self.loadAppViewData();
-                    }
-                    else if (selected === 'New crash occurence') {
-                        $("#single-target-condition-dropdown").css("visibility", "hidden");
-                        $('#alert-compare-value').css("visibility", "hidden");
-                    }
-                    else if (selected === 'Monthly data points' || selected === 'Hourly data points') {
-                        $("#single-target-condition-dropdown").clySelectSetSelection("reach threshold", "reach threshold");
-                        $("#alert-view .alert-compare-value-class").addClass("datapoint");
-                        setTimeout(function() {
-                            $("#single-target-condition-dropdown").clySelectSetItems([{value: 'reach threshold', name: 'reach threshold'}]);
-                        }, 10);
+        resetAlertCondition: function () {
+            this.$children[0].$data.editedObject.alertDataSubType = null;
```

Comment:
instead of using the this.$children[0] maybe you should give a ref to the child and access it via that ref.
this.refs.child.data